### PR TITLE
Add form directive

### PR DESCRIPTION
**Pull request by wytrych**

### CODE REVIEW

**Comment by wytrych** [comment]
@incuna/js Please review

File: web/app/scripts/src/directives/form.es.js [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+
+module.directive('surveysForm', [
+    FieldsetParser.componentName,
+    function (FieldsetParserService) {
+        return {
+            restrict: 'A',
+            scope: {
+                formStructure: '='
+            },
+            template: '<formly-form model="model" fields="fields"></formly-form>',
```
Comment (wytrych) [inline web/app/scripts/src/directives/form.es.js]: For some strange reason this errors when used as an attribute. I filed a bug https://github.com/formly-js/angular-formly/issues/708

File: web/app/index.html [hunk 2]
```diff
@@ -41,6 +41,10 @@
             <hr>
             <h5>Model:</h5>
             {{ form.model }}
+
+            <hr>
```
Comment (Minglee01) [inline web/app/index.html]: Does `<hr>` not have closing tags?

Comment (wytrych) [inline web/app/index.html]: Nope, it doesn't have any content

Comment (wytrych) [inline web/app/index.html]: https://developer.mozilla.org/en/docs/Web/HTML/Element/hr

**Comment by wytrych** [comment]
@incuna/js bump

**Pull request merged**